kube-manifest.sample.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: List
+items:
+  - apiVersion: v1
+    kind: Service
+    metadata:
+      name: zipkin
+    spec:
+      selector:
+        app: zipkin-stackdriver
+      ports:
+        - protocol: TCP
+          targetPort: 9411
+          port: 9411
+  - apiVersion: extensions/v1beta1
+    kind: Deployment
+    metadata:
+      name: zipkin-stackdriver
+      labels:
+        app: zipkin-stackdriver
+    spec:
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: zipkin-stackdriver
+        spec:
+          containers:
+            - name: zipkin-stackdriver
+              image: GoogleCloudPlatform/stackdriver-zipkin:latest
```

Comment:
I've hosted the docker image at b.gcr.io/stackdriver-trace-docker/zipkin-collector. We can reference that here.